external/storm-hdfs/src/test/java/org/apache/storm/hdfs/blobstore/BlobStoreTest.java

```diff
@@ -76,6 +76,7 @@
 
   @Before
   public void init() {
+    System.setProperty("test.build.data", "target/test/data");
     initializeConfigs();
     baseFile = new File("/tmp/blob-store-test-"+UUID.randomUUID());
```

Comment:
It breaks the blob store tests, and chasing down the root cause was getting to be a pain. At least writing test data to /tmp doesn't clutter up the source tree, but I would also like to avoid that as well. I figured we could address that separately.